tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -14,6 +22,20 @@ class TestAddCapacity(ManageTest):
     """
     Automates adding variable capacity to the cluster while IOs running
     """
+
+    @pytest.fixture(autouse=True)
+    def health_checker(self):
```

Comment:
If all it needs is health_check, I think we can close this PR since https://github.com/red-hat-storage/ocs-ci/pull/1656 this is merged.

Reply:
Thanks!